targets/targets.json

```diff
@@ -51,6 +51,10 @@
             "init-us-ticker-at-boot": {
                 "help": "Initialize the microsecond ticker at boot rather than on first use, and leave it initialized. This speeds up wait_us in particular.",
                 "value": false
+            },
+            "enable-floating-point": {
```

Comment:
I don't think this parameter belong in targets.json. I would define it as a config parameter in mbed-os/platform/mbed_lib.json instead.